en/lessons/specifics/ecto_basics.md

```diff
@@ -0,0 +1,196 @@
+---
+version: 1.3.0
+title: Ecto
+---
+
+Ecto allows you to query and store data into databases using its own query language.  It is typically used with relational databases such as PostgreSQL.
+
+If you've worked with an Object Relational Mapper (ORM) like ActiveRecord, Entity Framework, or Hibernate many of the concepts in Ecto will be similar.
```

Comment:
I would disagree with this personally. I think the language of the existing lesson is more fitting

> Ecto is an official Elixir project providing a database wrapper and integrated query language. With Ecto we’re able to create migrations, define schemas, insert and update records, and query them. (it doesnt have a section I can hyperlink to but scroll up just a bit [here](https://github.com/elixirschool/elixirschool/blob/master/en/lessons/specifics/ecto.md#setup) to see the original.

Reply:
To elaborate on my disagreement: this follows a repository pattern while AR (what I believe most people who come to us with the ruby-heavy background of Elixir learners in a lot of cases) disregards it in favor of abstraction that reads a little more nicely but has a lot of magic. I do not know the others mentioned, though.

Reply:
I agree with you that they are not the same.  However, I came at it from more of a high level perspective.  If we take Ecto and AR as an example-- they both:
* Provide a way to modify and create database schemas using migrations
* Provide a query language that will ultimately be translated to the database's native query language based on the adapter used.
* Take the result of the query and maps it to a native data type (object in AR, struct in ecto)

I'm fine with putting the existing lesson's explanation back though.